nav.yml

```
@@ -68,52 +65,12 @@ nav:
     - Policy Attachment: reference/policy-attachment.md
   - Enhancements:
     - Overview: geps/overview.md
-    - Implementable:
-      - geps/gep-3779/index.md
-      - geps/gep-3793/index.md
-      - geps/gep-3949/index.md
-    - Experimental:
-      - geps/gep-91/index.md
-      - geps/gep-1494/index.md
-      - geps/gep-1619/index.md
-      - geps/gep-1713/index.md
-      - geps/gep-1731/index.md
-      - geps/gep-1748/index.md
-      - geps/gep-1767/index.md
-      - geps/gep-3155/index.md
-      - geps/gep-3388/index.md
-      - geps/gep-3567/index.md
-    - Standard:
-      - geps/gep-709/index.md
-      - geps/gep-718/index.md
-      - geps/gep-724/index.md
-      - geps/gep-726/index.md
-      - geps/gep-746/index.md
-      - geps/gep-820/index.md
-      - geps/gep-851/index.md
-      - geps/gep-957/index.md
-      - geps/gep-995/index.md
-      - geps/gep-1016/index.md
-      - geps/gep-1294/index.md
-      - geps/gep-1323/index.md
-      - geps/gep-1364/index.md
-      - geps/gep-1686/index.md
-      - geps/gep-1709/index.md
-      - geps/gep-1742/index.md
-      - geps/gep-1762/index.md
-      - geps/gep-1867/index.md
-      - geps/gep-1897/index.md
-      - geps/gep-1911/index.md
-      - geps/gep-2162/index.md
-      - geps/gep-2257/index.md
-      - geps/gep-3171/index.md
-    - Memorandum:
-      - geps/gep-713/index.md
-      - geps/gep-917/index.md
-      - geps/gep-922/index.md
-      - geps/gep-1324/index.md
-      - geps/gep-2722/index.md
-      - geps/gep-2907/index.md
+    - Provisional: geps/landing/provisional.md
+    - Implementable: geps/landing/implementable.md
+    - Experimental: geps/landing/experimental.md
+    - Standard: geps/landing/standard.md
+    - Memorandum: geps/landing/memorandum.md
+    - List: geps/landing/tab.md
```

I think this should call out the states a bit separately, something like this? (Also reorders the states to sort them in decreasing order of stability).

```suggestion
    - Overview: geps/overview.md
    - List: geps/landing/tab.md
    - By State:
      - Standard: geps/landing/standard.md
      - Memorandum: geps/landing/memorandum.md
      - Experimental: geps/landing/experimental.md
      - Implementable: geps/landing/implementable.md
      - Provisional: geps/landing/provisional.md
```